.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,19 @@
+---
+name: Bug Report
+about: Create a report to help us improve
+
+---
+
+<!--
+YOUR ISSUE MAY BE CLOSED IF YOU DO NOT FOLLOW THIS TEMPLATE
+
+Consider searching for similar issues before submitting yours.
+-->
+
+**To Reproduce:** <!-- A simple set of reproduction steps or a MINIMAL reproducible example -->
```

Comment:
Steps to Reproduce sounds better imo

Reply:
I chose this wording instead considering it doesn't necessarily have to be a list of steps.

Reply:
Fair enough